rpcd/playbooks/roles/rpc_support/defaults/main.yml

```diff
@@ -70,6 +70,11 @@ holland_packages:
   - mariadb-client
   - xtrabackup
 
+holland_venv_enabled: False
+holland_venv_tag: untagged
```

Comment:
Needs to be set to rpc_release:
https://github.com/rcbops/rpc-openstack/blob/9174062e6efdbd35e0b06e25ac78e932e5d0633c/rpcd/playbooks/group_vars/all.yml#L19